src/endless/res/EndlessUsbTool.htm

```diff
@@ -307,9 +312,12 @@
                 
 					<div id="SecureBootReminder" class="RoundedCornersContainer">
 						<div id="ReminderTextContainer">
-                            <div class="ReminderTitle" id="ThankYouDualBootReminder">Restart your computer and select "Endless OS" from the menu.</div>
-                            <div class="ReminderTitle" id="ThankYouLiveReminder">To try Endless OS now, restart your computer.</div>
-                            <div class="ReminderTitle" id="ThankYouReflasherReminder">Now you're ready to install Endless OS on more computers.</div>
+                            <div class="ReminderTitle ModeNormal" id="ThankYouDualBootReminder">Restart your computer and select "Endless OS" from the menu.</div>
+                            <div class="ReminderTitle ModeCoding" id="ThankYouDualBootReminderCoding">Restart your computer and select "Endless Code" from the menu.</div>
```

Comment:
It comes from `/etc/os-release`: https://github.com/endlessm/ostree/blob/923e6dd80fcc15084295932502b2aeab4a96301f/src/libostree/ostree-sysroot-deploy.c#L1410

This is created in `base-files`, I think: `https://github.com/endlessm/base-files/blob/84f19f81c562ef58f7d956f8a393984422d818ab/src/os-release.in`

It's already patched by the ostree builder to include the build ID so you could conceivably arrange for it to be patched with the product name there: https://github.com/endlessm/eos-ostree-builder/blob/master/hooks/os/60-image_version

Maybe ask dsd or dbn?